v8pp/class.hpp

```diff
@@ -271,7 +271,8 @@ class class_
 	/// Set class member data
 	template<typename Attribute>
 	typename std::enable_if<
-		std::is_member_object_pointer<Attribute>::value, class_&>::type
+		std::is_member_object_pointer<Attribute>::value &&
+		!detail::is_const_member_pointer<Attribute>::value, class_&>::type
```

Comment:
Error occurs at the level of instantiation of &member_set<attribute_type>. I think that without a lovely `contexpr if` this can't be solved. Or maybe it can be solved with is_const_member_pointer specialization for member_set itself.